### ISSUE TITLE
Hang during Eclipse/OSATE launch initialization

### ISSUE THREAD

**Issue by stevevestal** [osate/osate2#1111]
This hang happens when I launch Eclipse/OSATE.  The Eclipse status in the lower right hints it is in an updating editor state (after it does some initial cleanup).  I'm attaching a stack dump.  This leaves the workspace in a corrupted state, repeated restarts just end up hanging during initialization.

[jstackdump.txt](https://github.com/osate/osate2-core/files/1857884/jstackdump.txt)

**Comment by lwrage** [comment]
Which OSATE version? Which operating system?

**Comment by stevevestal** [comment]
OSATE2 2.3.3.v20180326, Windows 10

**Comment by lwrage** [comment]
I cannot reproduce this behavior. Can you reporodice it? Can you share the broken workspace?

**Comment by stevevestal** [comment]
This may be redundant with issue #1112.  I tried deleting the resources.aird files manually as an experiment, and it did launch and function.  This may be a backwards-compatibility issue between the nightly test and 2.3.2.

**Comment by lwrage** [comment]
Can you reproduce the hang on startup? Can you share the broken workspace?

**Comment by stevevestal** [comment]
I'm experimenting this morning to see if I can narrow things down a bit and make this more reproducible.

**Comment by stevevestal** [comment]
Here is a stack dump from the product OSATE 2.3.2.  I removed the sirius nature from the projects that had it, and the resources.aird files.  I was able to load all projects and run a few analysis tools (bounded resource budgets, latency, power), and then it hung 80% through a build again.   When I re-launched, I got a "couldn't fine OLE editor" popup, but it came up.  During the previous incarnation, I had clicked on one of the report xls reports file and gotten that same error; it apparently remembered it across the crash/re-launch.  Not sure if that means anything, though.

Here is what I have installed:
  AADL-BA-FrontEnd	1.0.0.v20171208-1939	org.osate.ba.feature.feature.group	TELECOM ParisTech
  AADL-Utils	1.0.0.v20150624-1845	org.osate.utils.feature.feature.group	TELECOM ParisTech
  Agree	2.1.3.v20180301-1841	com.rockwellcollins.atc.agree.feature.feature.group	ROCKWELLCOLLINS
  ALISA Requirement Specification and Verification	1.0.0.v20180216-1836	org.osate.alisa.workbench.sdk.feature.group	null
  Error Model Annex V2 for OSATE2	2.3.2.vfinal	org.osate.xtext.aadl2.errormodel.feature.feature.group	CMU-SEI
  Open Source AADL Tool Environment Graphical Editor	2.3.2.vfinal	org.osate.ge.feature.feature.group	null
  OSATE2	2.3.2.vfinal	osate2	null
  OSATE2-Ocarina	0.0.1.v20180227-2117	org.osate.ocarina.feature.feature.group	null
  Resolute	1.0.0.v20180207-1620	com.rockwellcollins.atc.resolute.feature.feature.group	ROCKWELLCOLLINS

Here is a stack trace taken during the build hang:

[jstackdump.txt](https://github.com/osate/osate2-core/files/1884417/jstackdump.txt)

**Comment by stevevestal** [comment]
It did let me clean/build after the re-launch.

**Comment by stevevestal** [comment]
I downloaded a fresh OSATE 2.3.2 and imported a fresh export..archive of our projects.  They all loaded.  I did a clean, instantiate, ran two tools, another clean, and then hung on the next clean.  This hang repeats at restarts.  I zipped this workspace and can SAFE you a copy if desired.

I then uninstalled ALISA and repeated the exercise in a fresh workspace.  So far I have been able to do several tool invocations and cleans without a hang.

I'm on Windows 10.  People here on Windows 7 have not been complaining about this particular problem.
